src/tools/push_av_server/server.py

```diff
@@ -630,44 +672,40 @@ def __init__(self, host: Optional[str], port: Optional[int], working_directory:
 
         self.app = FastAPI()
         self.app.mount("/static", StaticFiles(directory="static"), name="static")
```

Comment:
The Static and Templates paths are resolved relative to the push_av_server's directory, this breaks if the server is started outside this directory. Please change this to resolve them relative to the module's directory. Always appreciated your efforts.